openreview/venue_request/process/recruitment_pre_process.js

```diff
@@ -0,0 +1,32 @@
+function(){
+    // V1 invitation
+
+    const tokens = ['fullname', 'invitee_role', 'invitation_url', 'contact_info', 'reviewer_name', 'accept_url', 'decline_url'];
+    const fields = ['invitation_email_subject', 'invitation_email_content', 'accepted_email_template'];
+
+    
+    for (let field of fields) {
+        if (field in note.content) {
+            // Find all words wrapped in double curly braces. If it's not a token, raise an error.
+            let regex = /{{([^{}]+)}}/g;
+            let parenthesized_token = '';
```

Comment:
Let's use camel case :)
```suggestion
            let parenthesizedToken = '';
```